helm/values-dev.yaml

```diff
@@ -6,26 +6,42 @@ microservice-chart:
     repository: pagopadcommonacr.azurecr.io/pagopaecommercewatchdogdeadletterfe
     tag: "latest"
     pullPolicy: Always
+  livenessProbe:
+    handlerType: httpGet
+    httpGet:
+      path: /
+      port: 3000
+    initialDelaySeconds: 10
```

Comment:
Maybe we can lower it to 5 seconds? I think the node server will start up in a few seconds, at worst we'll let the retry run